dashboards/Gamestudio-qs/README.md

```diff
@@ -0,0 +1,17 @@
+Name:
```

Comment:
We don't publish any files besides the core configuration files, could you get any important information in this README added to the description of the quickstart?